### ISSUE TITLE
Table "BLOBS" already exists when using Spring Content JPA with H2 database in multiple unit tests

### ISSUE THREAD

**Issue by DarioArena87** [paulcwarren/spring-content#2037]
**Describe the bug**
I'm using Spring Content JPA via the spring-content-jpa-boot-starter in a spring boot application and i'm creating some unit tests using H2 database (bad practice since the application will use PostgreSQL as a database but the inclusion of spring content is in an evaluation phase). 
When i run multiple tests that share the same application context i receive the error 
```
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "BLOBS" already exists; SQL statement:
CREATE TABLE BLOBS (id INTEGER GENERATED BY DEFAULT AS IDENTITY (START WITH 1), content BLOB, PRIMARY KEY ( id )) [42101-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:514) ~[h2-2.2.224.jar:2.2.224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489) ~[h2-2.2.224.jar:2.2.224]
	at org.h2.message.DbException.get(DbException.java:223) ~[h2-2.2.224.jar:2.2.224]
        ....
```
Looking at the schema-h2.sql that spring content autoconfiguration is using to initialize the database i see that the statement is not using the IF NOT EXISTS clause in table creation but i think it should solve the problem (like it is done in other scripts for "real" databases like PostgreSQL, MySQL or SQLServer).
Can the initialization script for the H2 database be modified to something like this?

```sql
CREATE TABLE IF NOT EXISTS BLOBS (id INTEGER GENERATED BY DEFAULT AS IDENTITY (START WITH 1), content BLOB, PRIMARY KEY ( id ))
ALTER TABLE BLOBS ALTER COLUMN id VARCHAR(36)
```

If there is nothing bad in modifying the initialization script to include an if not exists clause and there are more critical issues to resolve at the moment i can also do it myself and submit a PR (i've noticed the same issue is also in schema-oracle.sql)

**Comment by paulcwarren** [comment]
Hi @DarioArena87 , I think that was an oversight on my part.  If you have time a PR would be great.

**Comment by DarioArena87** [comment]
I added the clause for H2 database initialization in the submitted PR.
I'm not an expert in Oracle but from what i've found on the internet there seems not to be a similar statement and the closest one that i've found is using a procedure to drop the table if it exists and recreate it (which i don't particularly like because it drops eventual grants/permission given to the users).
Maybe if there's someone more skilled with Oracle he could suggest a method for creating the table BLOBS only if it does not exists.

**Comment by paulcwarren** [comment]
Thanks much

**Comment by paulcwarren** [comment]
Fixed by #2040
